monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -35,6 +35,13 @@ def base_package_name():
     # Usual values are 'vulnerability' or 'brute_force'
     EXPLOIT_TYPE = ExploitType.VULNERABILITY
 
+    # Specifies whether a machine, on which the exploit was successful, should be added to the set of exploited
+    # machines. This would then prevent any other exploits from being attempted on it.
+    # Sample use case - Zerologon exploiter:
+    #     Exploited machine gives us useful credentials which can be used, but machine isn't compromised by Zerologon
+    #     on its own. Some other exploit using PTH needs to be exploit it.
```

Comment:
Grammar? What does `Some other exploit using PTH needs to be exploit it.` mean?

Reply:
Sorry, the "be" shouldn't be there